CHANGELOG.md

```diff
@@ -18,6 +18,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## 📚 Documentation -->
 
+# [0.26.3] (unreleased) - 2024-12-dd
```

Comment:
The main purpose of this PR is to provide a place for review of the CHANGELOG text below. As described on my comment on the PR, this line will be updated before release with the date (and no "unreleased").